react/src/util/opReturn.ts

```diff
@@ -85,21 +98,24 @@ function getDataPushdata(data: string, disablePaymentId=false) {
 
 export function parseOpReturnProps(
```

Comment:
what does this function do? Looks like it returns a hex string that is ready for `op_return_raw` param?

would help to have some comments about the function's inputs, outputs, mb ref the spec. `parseOpReturnProps` is confusing since it does not seem to be doing this. 

mb `encodePaybuttonOpReturnRaw` or similar. Or mb I am misreading it. 

It's unclear.

Reply:
It encodes the `op-return=""` props the user has passed to the paybutton.

Indeed `encode` here is better than `parse`; so I've changed to `encodeOpReturnProps` which refers to the function input; the `op_return_raw` ready hex is the output, which is explained in the comments above the function declaration (also added one more comment at the end). Let me know what you think